### PR TITLE
MBS-10392 Convert Add Series edit to React

### PR DESCRIPTION
**Pull request by anirudhjain75**

### CODE REVIEW

File: root/edit/details/AddSeries.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    +series: SeriesT,
+    +type: SeriesTypeT,
+  },
+};
+
+type Props = {
+  edit: AddSeriesEditProps,
+};
+
+const AddSeries = ({edit}: Props) => {
```
Comment (reosarevok) [inline root/edit/details/AddSeries.js]: Wouldn't it be easier to do 
```suggestion
const AddSeries = ({edit}: {edit: AddSeriesEditProps}) => {
```
here and avoid the second Props above?

File: root/edit/details/AddSeries.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          {edit.display_data.comment ? (
+            <tr>
+              <th>{addColonText(l('Disambiguation'))}</th>
+              <td>{edit.display_data.comment}</td>
+            </tr>
+          ) : null}
+          {edit.display_data.type ? (
+            <tr>
+              <th>{addColonText(l('Type'))}</th>
+              <td>{edit.display_data.type.name}</td>
```
Comment (reosarevok) [inline root/edit/details/AddSeries.js]: We should probably translate this, even if it wasn't translated originally for some reason.

File: root/edit/details/AddSeries.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            </tr>
+          ) : null}
+          {edit.display_data.ordering_type ? (
+            <tr>
+              <th>{addColonText(l('Ordering Type'))}</th>
+              <td>{edit.display_data.ordering_type.name}</td>
```
Comment (reosarevok) [inline root/edit/details/AddSeries.js]: I think this should be translatable too, but I'm not sure how exactly. @mwiencek, would this be l_attributes with ordering_type or something? (there's l_name but we shouldn't use that).

File: root/edit/details/AddSeries.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    <>
+      <table className="details">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Series'))}</th>
+            <td><EntityLink entity={edit.display_data.series} /></td>
+          </tr>
+        </tbody>
+      </table>
+      <table className="details add-series">
```
Comment (reosarevok) [inline root/edit/details/AddSeries.js]: I know this is in the original code, but I can't figure out why we have these two as separate tables...

File: root/edit/details/AddSeries.js (hunk 2 above)
Comment (reosarevok) [reply root/edit/details/AddSeries.js]: I think you'd want:

```suggestion
              <td>{lp_attributes(edit.display_data.type.name, 'series_type')}</td>
```

File: root/edit/details/AddSeries.js (hunk 3 above)
Comment (reosarevok) [reply root/edit/details/AddSeries.js]: I think:

```suggestion
              <td>
                {lp_attributes(
                  edit.display_data.ordering_type.name,
                  'series_ordering_type',
                )}
              </td>
```

File: root/edit/details/AddSeries.js (hunk 4 above)
Comment (reosarevok) [reply root/edit/details/AddSeries.js]: On second thought, you can leave this be for now and we can fix them all in one commit later.

File: root/edit/details/AddSeries.js (hunk 3 above)
Comment (anirudhjain75) [reply root/edit/details/AddSeries.js]: thanks :)

File: root/edit/details/AddSeries.js (hunk 4 above)
Comment (anirudhjain75) [reply root/edit/details/AddSeries.js]: Okay !

File: root/edit/details/AddSeries.js (hunk 2 above)
Comment (anirudhjain75) [reply root/edit/details/AddSeries.js]: Thanks

**Comment by mwiencek** [comment]
This would crash trying to display edits where the series was later deleted. I fixed that and some minor issues with the Flow types. See https://github.com/metabrainz/musicbrainz-server/compare/890c5a11eb1beec038004845f5b35164693d1e3e..89d79a101455933e7845df7d3cd32c5194a96f9a

**Pull request merged**